scripts/vm-nat-testing/faucet.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+cd /opt/hopr || exit 1
+
+HOPR_ENVIRONMENT_ID=hardhat-localhost yarn workspace @hoprnet/hopr-ethereum \
+	hardhat faucet --use-local-identities --identity-directory /var/hopr/identities/ --identity-prefix local \
```

Comment:
```suggestion
	hardhat faucet --use-local-identities --identity-directory /var/hopr/identities/ --identity-prefix "local" \
```
Isn't "local" a keyword in shell scripts?

Reply:
it is, but only as part of the declaration of a variable.